src/main/java/com/exadel/discountwebapp/history/vo/HistoryResponseVO.java

```diff
@@ -0,0 +1,26 @@
+package com.exadel.discountwebapp.history.vo;
+
+import com.exadel.discountwebapp.history.enums.HistoryEnum;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.javers.core.diff.changetype.PropertyChangeType;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Data
+@SuperBuilder
+@NoArgsConstructor
+@Setter
+@Getter
+@Builder
```

Comment:
we have 2 builder on this class: @SuperBuilder and @Builder

Reply:
removed @ builder